sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
 
+### New Features
+- Support setting blob sas permission with a raw string.
```

Comment:
Is this accurate and do we need to repeat it for each service?

Reply:
updated